scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -1408,6 +1422,9 @@ def yen(
     If multiple valid solutions are possible, output may vary with SciPy and
     Python version.
 
+    The algorithmic complexity for a graph with V number of vertices and E
+    number of edges is O(K*V*(E + V*logV)).
```

Comment:
What is K here? Is that the arbitrary constant in the big-O notation?

Reply:
`K` is the input argument of `yen`'s algorithm. `yen`'s algorithm finds the K-shortest paths.